website/docs/cloud-docs/vcs/troubleshooting.mdx

```diff
@@ -8,6 +8,19 @@ description: >-
 
 This page collects solutions to the most common problems our users encounter with VCS integration in Terraform Cloud.
 
+## Azure DevOps
```

Comment:
This may need to be copied to the TFE equivalent as well.

https://terraform-docs-common-git-ggambetti-ado-status-checks-hashicorp.vercel.app/terraform/enterprise/vcs/troubleshooting

Reply:
https://github.com/hashicorp/ptfe-releases/blob/main/website/docs/enterprise/vcs/troubleshooting.mdx

Reply:
Ack.

Reply:
I've got the backport drafted: https://github.com/hashicorp/ptfe-releases/pull/953